virt/vm_networking/virt-networking-overview.adoc

```diff
@@ -0,0 +1,116 @@
+
+The Data Plane Development Kit (DPDK) provides a set of libraries and drivers for fast packet processing. You can configure clusters and virtual machines (VMs) to run DPDK workloads over SR-IOV networks.
+
+xref:../../virt/vm_networking/virt-dedicated-network-live-migration.adoc#virt-dedicated-network-live-migration[Configuring a dedicated network for live migration]::
+
+You can configure a dedicated xref:../../virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc#virt-connecting-vm-to-linux-bridge[Multus network] for live migration. A dedicated network minimizes the effects of network saturation on tenant workloads during live migration.
+
+xref:../../virt/vm_networking/virt-accessing-vm-secondary-network-fqdn.adoc#virt-accessing-vm-secondary-network-fqdn[Accessing a virtual machine by using the cluster FQDN]::
+
+You can access a virtual machine (VM) that is attached to a secondary network interface from outside the cluster by using its fully qualified domain name (FQDN).
+
+xref:../../virt/vm_networking/virt-configuring-viewing-ips-for-vms.adoc#virt-configuring-viewing-ips-for-vms[Configuring and viewing IP addresses]::
+
+You can configure an IP address of a secondary network interface when you create a virtual machine (VM). The IP address is provisioned with cloud-init. You can view the IP address of a VM by using the {product-title} web console or the command line. The network information is collected by the QEMU guest agent.
+
+[id="vm-as-a-service"]
+== Configuring VM as a service
+
+xref:../../virt/vm_networking/virt-exposing-vm-with-service.adoc#virt-exposing-vm-with-service[Exposing a virtual machine by using a service]::
+
+You can expose a virtual machine within the cluster or outside the cluster by creating a `Service` object. For on-premise clusters, you can configure a load balancing service by using the MetalLB Operator. You can xref:../../networking/metallb/metallb-operator-install.adoc#metallb-operator-install[install the MetalLB Operator] by using the {product-title} web console or the CLI.
+
+[id="service-mesh-integration"]
+== Integrating with OpenShift Service Mesh
+xref:../../virt/vm_networking/virt-connecting-vm-to-service-mesh.adoc#virt-connecting-vm-to-service-mesh[Connecting a virtual machine to a service mesh]::
+
+{VirtProductName} is integrated with OpenShift Service Mesh. You can monitor, visualize, and control traffic between pods and virtual machines.
+
+[id="managing-mac-address-pools"]
+== Managing MAC address pools
+xref:../../virt/vm_networking/virt-using-mac-address-pool-for-vms.adoc#virt-using-mac-address-pool-for-vms[Managing MAC address pools for network interfaces]::
+
+The KubeMacPool component allocates MAC addresses for virtual machine (VM) network interfaces from a shared MAC address pool. This ensures that each network interface is assigned a unique MAC address. Virtual machine instances created from that VM retain the assigned MAC address across reboots.
```

Comment:
`Virtual machine instances created from that VM...`
I think this sentence is confusing, as it implies that one VM object can result with multiple VMIs, which is not true.
Maybe the phrasing `A virtual machine instance created from that VM...` would be more accurate.
If you agree, then take into account that this should also be changed in https://69551--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/vm_networking/virt-using-mac-address-pool-for-vms#virt-using-mac-address-pool-for-vms.